api/v2/checluster_types.go

```diff
@@ -90,6 +90,16 @@ type CheClusterDevEnvironments struct {
 	// These default components are meant to be used when a Devfile, that does not contain any components.
 	// +optional
 	DefaultComponents []devfile.Component `json:"defaultComponents,omitempty"`
+	// Idle timeout for workspaces, for example: 15m, 30m, 1h.
+	// This timeout is the duration after which a workspace will be idled if there is no activity.
+	// If this value is not provided, workspaces will not idle.
+	// +optional
+	SecondsOfInactivityBeforeIdling string `json:"secondsOfInactivityBeforeIdling,omitempty"`
+	// Run timeout for workspaces, for example: 15m, 30m, 1h.
```

Comment:
BWT, There is some inconsistency between field name (starts with `Seconds...`) and description (we mention time interval like `15m`).  If everything about seconds, then we need to mention duration in minutes or hours? 
Should it be like
`SecondsOfRunBeforeIdling *int32 `json:"secondsOfRunBeforeIdling,omitempty"` ?

Reply:
Oops, yes my mistake. It should be `SecondsOfRunBeforeIdling *int32`, and the comments must be updated